src/pages/home/index.tsx

```diff
@@ -8,7 +9,9 @@ export function Home(): JSX.Element {
   return (
     <Flex h="100vh" direction="column">
       <Header />
-      <Landing />
+      <Center h="100%">
+        <Landing />
+      </Center>
```

Comment:
The `<Center></Center>` should be in the `<Landing />`.

Reply:
My rational was not to assume layout in the `<Landing/>` component itself since it might be used in other ways.